src/app/stage/stage/stage.component.html

```diff
@@ -24,7 +24,7 @@
     aria-label="Presets"
     mat-icon-button
     matTooltip="Presets"
-    matBadge="{{ presets.length }}"
+    [matBadge]="presets.length | number"
```

Comment:
Not sure, if it good approach 😬

Reply:
Ivy... ok, let it be like that for now ;)